docs-src/concepts/what-is-an-activity.md

```diff
@@ -11,7 +11,7 @@ tags:
 In day-to-day conversations, the term _Activity_ frequently denotes either an [Activity Definition](/concepts/what-is-an-activity-definition), an [Activity Type](/concepts/what-is-an-activity-type), or an [Activity Execution](/concepts/what-is-an-activity-execution).
 Temporal documentation aims to be explicit and differentiate between them.
 
-An Activity is a normal function or object method that executes a single, well-defined action (either short or long running), such as calling another service, transcoding a media file, or sending an email message.
+Temporal recommends that Activities are idempotent. Activities are idempotent if multiple applications of that operation do not change the state of the system beyond the initial application.
```

Comment:
Remove and revert this, as it is covered in the "Idempotency section" of the Activity Definition concept - and these files are back to back in the guide.